### PR TITLE
Adds contributors screen

### PR DESCRIPTION
**Pull request by ccosnean**

### CODE REVIEW

File: lib/modules/contributors/services/github_service.dart [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import 'open_contributors_cache_manager.dart';
+
+
+class InvalidResponseError extends StateError {
+  InvalidResponseError(String msg) : super(msg);
+}
+
+class GithubService {
+  static const BASE_URL = 'https://api.github.com';
+  static const CREDENTIALS = '?client_id=2a73a1450d7b195e0d62&client_secret=29e82f1e6ad8a081d47fcf0a338b772b98166388';
```
Comment (stpopa) [inline lib/modules/contributors/services/github_service.dart:14]: are you sure you want those here?

File: lib/modules/contributors/services/github_service.dart [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import 'dart:async';
+import 'dart:convert' as convert;
+import 'package:http/http.dart' as http;
+import '../models/contributor.dart';
+import '../models/user.dart';
+import 'open_contributors_cache_manager.dart';
+
+
```
Comment (stpopa) [inline lib/modules/contributors/services/github_service.dart]: Extra line?

File: lib/modules/contributors/contributors_screen.dart [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import 'contributors_body.dart';
+import 'models/user.dart';
+import '../../themes/contributors_theme.dart';
+
+class ContributorsScreen extends StatelessWidget {
+  final String repository;
+
+  ContributorsScreen({this.repository});
+
+
```
Comment (stpopa) [inline lib/modules/contributors/contributors_screen.dart]: Extra line?

File: lib/modules/contributors/contributors_body.dart [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  }
+
+  ContributorAccessory _accessoryFor(int index) {
+    switch (index) {
+      case 0:
+        return ContributorAccessory.first;
+      case 1:
+        return ContributorAccessory.top3;
+      case 2:
+        return ContributorAccessory.top3;
```
Comment (stpopa) [inline lib/modules/contributors/contributors_body.dart]: Logic mistake? if not you can combine case 1 with case 2

File: lib/modules/contributors/services/github_service.dart (hunk 1 above)
Comment (ccosneanu) [reply lib/modules/contributors/services/github_service.dart:14]: For now yes ), later i will make a package out of it and require it as a param.

Comment (stpopa) [reply lib/modules/contributors/services/github_service.dart:14]: tumblr_nbv8jkKpnJ1tc258so2_r1_1280

**Review by stpopa — APPROVED** [review]

**Pull request merged**